packages/mds-transactions-service/@types/index.ts

```diff
@@ -116,13 +116,11 @@ export interface TransactionService {
   addTransactionOperation: (operation: TransactionOperationDomainCreateModel) => TransactionOperationDomainModel
   // read back operations for a transaction
   // if auth token has a provider_id, it must match the provider_id in the transaction
-  getTransactionOperations: (
-    transaction_id: TransactionDomainModel['transaction_id']
-  ) => TransactionOperationDomainModel[]
+  getTransactionOperations: () => TransactionOperationDomainModel[]
```

Comment:
So, we _always_ get _all_ operations for _all_ transactions now? That seems odd.

Reply:
derp.  my bad.  dumb!